### PR TITLE
create incuna normalise file

### PR DESCRIPTION
**Pull request by hippogriffic**
for sass we use on all our projects, after a slack discussion with the sass ladies

let me know if you think there's something else that could go in here.

I went with 1.4 for line height because I feel like I've used that most often in projects, but will happily change it if people disagree.

@incuna/frontend please review

### CODE REVIEW

File: incuna-sass/mixins/_incuna-normalise.sass [hunk 1]
```diff
@@ -0,0 +1,13 @@
+// Basic styles that we end up adding to all of our projects
```
Comment (pandalion) [inline incuna-sass/mixins/_incuna-normalise.sass]: probably american but maybe call incuna-normalize.sass 

this is a good idea :D

**Comment by maciejpi** [comment]
I'm good with that

**Comment by pandalion** [comment]
Looks good! (I know you're moving from mixins now to the root)

**Comment by hippogriffic** [comment]
@pandalion fixed

**Comment by henrahmagix** [comment]
> I went with 1.4 for line height because I feel like I've used that most often in projects, but will happily change it if people disagree.

👍  for me

**Comment by grahamgilchrist** [comment]
Looks good 👍 

Since there is another `_normalise.sass`, maybe a description somewhere (comments in the files or in the readme) about the difference and when you would want to use either?

**Comment by hippogriffic** [comment]
@incuna/frontend can this be merged?

File: incuna-sass/_incuna-normalize.sass [hunk 2]
```diff
@@ -0,0 +1,13 @@
+// Basic styles that we end up adding to all of our projects
+
+body
+    *
+        box-sizing: border-box
+        line-height: 1.4
```
Comment (perry) [inline incuna-sass/_incuna-normalize.sass]: I don't think you should set a line-height on all elements like this, just set it on the `body` as other elements will inherit it.

Comment (perry) [inline incuna-sass/_incuna-normalize.sass]: I think it will be fine, just feels wrong to me :-)

Comment (henrahmagix) [inline incuna-sass/_incuna-normalize.sass]: Yes, I agree: `line-height` is better controlled via inheritance

Comment (hippogriffic) [inline incuna-sass/_incuna-normalize.sass]: yeah that's fair

**Comment by hippogriffic** [comment]
@henrahmagix @perry done :)

**Comment by perry** [comment]
Can I press the green button :D?

**Comment by hippogriffic** [comment]
of course you can, in fact please do :D

**Comment by perry** [comment]
✨

**Comment by hippogriffic** [comment]
thanks :)

**Pull request merged**